idea/idea-native/src/org/jetbrains/kotlin/ide/konan/NativePlatformKindResolution.kt

```diff
@@ -183,6 +214,6 @@ class NativeLibraryInfo(project: Project, library: Library, root: File) : Librar
     override fun toString() = "Native" + super.toString()
 
     companion object {
-        val NATIVE_LIBRARY_CAPABILITY = ModuleDescriptor.Capability<KonanLibrary>("KonanLibrary")
+        val NATIVE_LIBRARY_CAPABILITY = ModuleDescriptor.Capability<KotlinLibrary>("KotlinLibrary")
```

Comment:
Maybe keep "KotlinNativeLibrary" as the name of capability? "KotlinLibrary" can be misunderstood.